api/sensor/samples/test_set.js

```diff
@@ -0,0 +1,20 @@
+console.log('Code start!');
+
+var sensorManager = require("../build/Release/sensor-api");
```

Comment:
Other sample/system applications does not use complicated API path like this.
Furthermore, ```sensorManager``` is the internal module name of ANT framework, not API name.
You should use ```sensorApi``` as the API object's name instead of ```sensorManager```.

You need to change this code to the following form.
```
var ant_api_dir = process.env.ANT_BIN_DIR + "/api/";
var sensorApi = require(ant_api_dir + "sensor-api");
```